docs/howtos/DataPlane Generation - DPG/07tcgcTypes.mdx

```diff
@@ -1040,13 +1044,16 @@ Currently we only support HTTP service parameters.
 This represents an HTTP query parameter.
 
 ```ts
-export type CollectionFormat = "multi" | "csv" | "ssv" | "tsv" | "pipes";
+export type CollectionFormat = "multi" | "csv" | "ssv" | "tsv" | "pipes" | "simple" | "form";
```

Comment:
nit, could we have a similar list as path? I am not sure whether I know what's for `simple` and `form`.

```
  // - simple: no separator
  // - label: . as separator
  // - matrix: ; as separator
  // - fragment: # as separator
  // - path: / as separator
```

Reply:
added, `simple` is same as `csv` and `form` is same as `multi`